google/cloud/storage/emulator/tests/test_gcs.py

```diff
@@ -415,6 +418,160 @@ def test_update_and_patch(self):
 
 
 class TestHolder(unittest.TestCase):
+    def test_init_resumable_rest_incorrect_usage(self):
+        bucket_metadata = json.dumps({"name": "bucket-test"})
+        environ = create_environ(
+            base_url="http://localhost:8080",
+            content_length=len(bucket_metadata),
+            data=bucket_metadata,
+            content_type="application/json",
+            method="GET",
```

Comment:
Question: these are all `GET` requests, but you should be testing stuff for `POST`?

Reply:
Whoops, method is ignored in tests because I only need the request object to pass to the underlying `init_resumable_rest()` method call. Addressing issue. Ty